DataFormats/SiPixelDetId/interface/PixelChannelIdentifier.h

```diff
@@ -2,6 +2,7 @@
 #define DATAFORMATS_PIXELCHANMNELIDENTIFIER_H
 
 #include <utility>
+#include <cstdint>
 
 namespace pixelchanelidentifierimpl {
```

Comment:
By the way, should this be `pixelchannelidentifierimpl` (with two `n`) ?

Reply:
I think so :)